app/Http/Controllers/AlbumController.php

```diff
@@ -10,9 +10,18 @@
 
 class AlbumController extends Controller
 {
-    public function index(Request $request, $id)
+    public function index(Request $request)
     {
-        gallery($request, $id);
+        // Should be replaced by the user id of the logged user
+        $myAlbums = Album::where('id_user', '=', 1)->get();
+
+        // Should find a command to get list of sharedAlbuns
```

Comment:
```suggestion
        // Should find a command to get list of sharedAlbums
```
Typo ;)

Reply:
Thanks for the review, it's super relevant !
Solved here eee8906a39519d58e0e7eb4c7b88e800ff02524a !